products/cakephp.md

```diff
@@ -19,222 +18,49 @@ auto:
-    latestReleaseDate: 2016-03-13
-    link: https://bakery.cakephp.org/2016/03/13/cakephp_2613_2711_282_3017_3112_325_released.html
-
--   releaseCycle: "2.6"
-    latest: "2.6.13"
-    eol: true
-    support: false
-    releaseDate: 2014-12-23
-    latestReleaseDate: 2016-03-13
-    link: https://bakery.cakephp.org/2016/03/13/cakephp_2613_2711_282_3017_3112_325_released.html
 
--   releaseCycle: "2.5"
-    latest: "2.5.9"
-    eol: true
-    support: false
-    releaseDate: 2014-05-12
-    latestReleaseDate: 2015-08-06
-    link: https://bakery.cakephp.org/2015/08/06/cakephp_2_5_9_2_6_10_2_7_2_released.html
-
--   releaseCycle: "2.4"
-    latest: "2.4.10"
-    eol: true
-    support: false
-    releaseDate: 2013-08-30
-    latestReleaseDate: 2014-05-17
-    link: https://bakery.cakephp.org/2014/05/18/CakePHP-2-4-10-and-2-5-1-released.html
-
--   releaseCycle: "2.3"
-    latest: "2.3.10"
-    eol: true
-    support: false
-    releaseDate: 2013-01-28
-    latestReleaseDate: 2013-08-30
-
--   releaseCycle: "2.2"
-    latest: "2.2.9"
-    eol: true
-    support: false
-    releaseDate: 2012-07-01
-    latestReleaseDate: 2013-07-17
-    link: https://bakery.cakephp.org/2012/07/14/Security-Release-CakePHP-2-1-5-2-2-1.html
-
--   releaseCycle: "2.1"
-    latest: "2.1.5"
-    eol: true
-    support: false
-    releaseDate: 2012-03-04
-    latestReleaseDate: 2012-07-14
-    link: https://bakery.cakephp.org/2012/07/14/Security-Release-CakePHP-2-1-5-2-2-1.html
-
--   releaseCycle: "2.0"
-    latest: "2.0.6"
-    eol: true
-    support: false
-    releaseDate: 2011-10-16
-    latestReleaseDate: 2012-02-05
-    link: https://bakery.cakephp.org/2012/02/06/CakePHP-2-0-6-released.html
-
--   releaseCycle: "1.3"
-    latest: "1.3.21"
-    eol: true
-    support: false
+-   releaseCycle: "1"
     releaseDate: 2010-04-25
+    support: 2015-10-31
+    eol: 2015-10-31
+    latest: "1.3.21"
     latestReleaseDate: 2015-10-31
-    link: https://bakery.cakephp.org/2015/11/01/cakephp_1_3_21_released.html
 
 ---
 
-> [CakePHP](https://cakephp.org/) is a free & open source PHP web development framework.  It follows the model–view–controller (MVC) approach and is written in PHP, modeled after the concepts of Ruby on Rails, and distributed under the [MIT License](https://en.wikipedia.org/wiki/MIT_License).
+> [CakePHP](https://cakephp.org/) is a free and open source PHP web development framework. It
+> follows the model–view–controller (MVC) approach and is written in PHP, modeled after the concepts
+> of Ruby on Rails.
 
-Have a look at the [Bakery](https://bakery.cakephp.org/) for fresh [releases](https://bakery.cakephp.org/categories/release.html).
+CakePHP follows [Semantic Versioning](https://book.cakephp.org/4/en/release-policy.html).
+Support and EOL policy for 1.x [was not clearly defined](https://stackoverflow.com/questions/20162089/cakephp-1-x-end-of-support-date).
+and [the 1.x cycle ended with 1.3.21](https://github.com/cakephp/cakephp/releases/tag/1.3.21).
```

Comment:
Not sure we need to mention that as a text (see my comment about structured data).